Makefile

```diff
@@ -5,6 +5,15 @@ HOSTNAME ?= $(shell hostname)
 # HOSTNAME is the name of the physical host
 export HOSTNAME := $(HOSTNAME)
 
+# Exposed ports to the host (default remains in container)
+# These each need to be separately tested before being enabled, so hard coded for now
```

Comment:
This comment seems outdated, except for flannel

Reply:
Fixed.